src/assimp/assimp.cpp

```diff
@@ -717,11 +785,14 @@ vsg::ref_ptr<vsg::Node> SceneConverter::visit(const aiScene* in_scene, vsg::ref_
     scene = in_scene;
     options = in_options;
     discardEmptyNodes = vsg::value<bool>(true, assimp::discard_empty_nodes, options);
+    externalTextures = vsg::value<bool>(false, assimp::external_textures, options);
+    externalTextureFormat = vsg::value<TextureFormat>(TextureFormat::native, assimp::external_texture_format, options);
 
     std::string name = scene->mName.C_Str();
 
     if (options) sharedObjects = options->sharedObjects;
     if (!sharedObjects) sharedObjects = vsg::SharedObjects::create();
+    if (!externalObjects) externalObjects = vsg::External::create();
```

Comment:
This looks like a vsg::External will always be created even when it's not required.  I would have thought a if (externalTextures && !externalObjects) would be more appropriate.

I also wonder if vsg::Options might benefit from having externalObjects member like it has sharedObjects member, though this obviously requires wider changes.

Reply:
> This looks like a vsg::External will always be created even when it's not required. I would have thought a if (externalTextures && !externalObjects) would be more appropriate.
OK I can change that, yes

Reply:
I changed that